flygym/fly.py

```diff
@@ -922,6 +887,64 @@ def _add_force_sensors(self):
 
         return force_sensors
 
+    def _add_joint_actuators(self, gain, forcerange):
+        # if self control is "motor" check that the gain is not provided
+        if self.control == "motor" and gain is not None:
+            # print warning
+            logging.warning(
+                "Motor control is selected, the gain parameter will not be used"
+            )
+
+        ## Need to deal with the kp, force range, and neck actuators
+        if not type(gain) == list:
```

Comment:
In general type checking should be done with `isinstance(variable, type)`. Here: `if not isinstance(gain, list)`. Same below. See https://stackoverflow.com/a/73596548/13560623.